test/metric_value_benchmark/ecs_daemon_base_test_runner.go

```diff
@@ -0,0 +1,103 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build linux && integration
+// +build linux,integration
+
+package metric_value_benchmark
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/aws/amazon-cloudwatch-agent-test/environment"
+	"github.com/aws/amazon-cloudwatch-agent-test/test"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/metric"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/status"
+)
+
+type IECSTestRunner interface {
```

Comment:
This share a lots with `basedTestRunner`. Should we create a common interface and `runStrategy` can be a part of it

Reply:
Yep this was intentional to split PRs. 1) needed to make it work for ecs to even know what I'm going to do  & 2) it was probably going to be extra line changes in this PR and take longer to make sure nothing breaks. I plan to merge this with a shared base runner with injectable structs